apigee_edge.routing.yml

```
@@ -190,3 +190,15 @@ apigee_edge.export_analytics.csv:
   requirements:
     _csrf_token: 'TRUE'
     _custom_access: '\Drupal\apigee_edge\Controller\ExportAnalyticsController::access'
+
+apigee_edge.redirect.developer_app.add_form_for_developer:
+  path: '/user/{entity_slug}/create-app'
+  defaults:
+    _controller: '\Drupal\apigee_edge\Controller\RedirectController::manageRedirect'
+    redirect_route: 'entity.developer_app.add_form_for_developer'
+  options:
+    parameters:
+      entity_slug:
+        type: entity:user
+  requirements:
+    _permission: 'access content'
```

I just noticed that if users have hardcoded any links in menus (using tokens or any other method) then the legacy links will not respect the original route access permissions because of this "access content" requirement (it will lead them to perhaps seeing a link that ultimately will give a 403 error if they don't have the necessary permissions).
I've added a suggestion of how we could rely on the original access controller instead:

```suggestion
    entity_slug: '\d+'
    _custom_access: '\Drupal\apigee_edge\Controller\RedirectController::createDeveloperAppAccess'
```
 Note: suggestion continues on the `RedirectController` class, and I've only added it for the `/user/{entity_slug}/create-app` route, it would need to be implemented on the other legacy routes as well, if you think this is a good approach.